### PR TITLE
[Compute] vm/vmss create: Encryption of cache & data-in-transit for OS/Data disks and temp disks for VM & VMSS

### LINKED ISSUE #13559: Cli support for Encryption of cache & data-in-transit for OS/Data disks and temp disks for VM & VMSS
**Resource Provider**
Compute Resource Provider

**Description of Feature or Work Requested**
We are introducing the encryption of the following:

1.	Encryption at rest for temp disks
2.	Encryption at rest for cache of OS and data disks
3.	Encryption of data-in-transit of OS and data disks

The above encryption features can be enabled by setting a new property EncryptionAtHost under securityProfile of VMs/VMSSs using the API version 2020-06-01 and above.

"securityProfile": { "encryptionAtHost": "true" }

You can find more details here: https://github.com/ramankumarlive/manageddisksendtoendencryptionpreview

**Minimum API Version Required**
2020-06-01 and above

**Swagger Link**
https://github.com/Azure/azure-rest-api-specs/pull/9366

**Target Date**
06/30/2020

it will help us prioritize your request against other requests. --->

### PR DESCRIPTION
**Pull request by qwordy**
**Description**  

Resolve #13559 
Add a new parameter for `vm create` and `vmss create`
```
--encryption-at-host
Enable Host Encryption for the VM or VMSS. This will enable the encryption for all the disks including Resource/Temp disk at host itself.  Allowed values: false, true.
```

Upgrate VMSS API version to 2020-06-01

**Testing Guide**  

```
az vm create --encryption-at-host
az vmss create --encryption-at-host
```

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Compute

**Comment by Juliehzl** [comment]
Approved for API Profile part.

File: src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py [hunk 1]
```diff
@@ -3375,6 +3375,24 @@ def test_vm_disk_encryption_e2e(self, resource_group, resource_group_location):
...
+        self.kwargs.update({
+            'vm': 'vm1',
+            'vmss': 'vmss1'
+        })
+
+        self.cmd('vm create -g {rg} -n {vm} --image centos --generate-ssh-keys --nsg-rule NONE --encryption-at-host')
+        self.cmd('vm show -g {rg} -n {vm}', checks=[
+            self.check('securityProfile.encryptionAtHost', True)
+        ])
+        self.cmd('vmss create -g {rg} -n {vmss} --image centos --generate-ssh-keys --encryption-at-host')
```
Comment (fengzhou-msft) [inline src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py:3391]: can you add `--admin-username` in `vm/vmss create`?

**Comment by qwordy** [comment]
> 
> 
> Add `--admin-username` in `vm/vmss create` before fixing #14265

Added

**Pull request merged**